src/uu/ls/src/ls.rs

```diff
@@ -954,6 +965,11 @@ impl Config {
             None
         };
 
+        let dired = options.get_flag(options::DIRED);
+        if dired && format != Format::Long {
+            crash!(1, "{}", LsError::ConflictingArgumentDired());
```

Comment:
I really want to remove exits and panic as much as possible, because they make it harder to embed uutils (in nushell, for example). Can we return an error here instead?